### DISCUSSION TITLE
Fixed-Wing VTOL Application

### DISCUSSION THREAD

**Discussion by Devin-007** [byuflowlab/FLOWUnsteady#73] [Q&A]
The tutorial provided is for a tilt-wing VTOL aircraft (Vahana), but would it be possible to simulate a fixed-wing VTOL as well using this software? If so, what resources should I look for to find out how to implement that with different rotor configurations? My goal is to set up a few different configurations of both wings and rotors, and compare them to find what performs the best. Any help would be greatly appreciated. Thanks!

**Comment by EdoAlvarezR** [comment]
We have added a series of tutorials to the docs, which hopefully will be useful to you: https://flow.byu.edu/FLOWUnsteady

**Reply by Devin-007** [reply]
Thank you for the speedy response. The VTOL application tutorial is for a tilt-wing vehicle, how would I go about changing that to fixed-wing? I'm looking to have the props facing upwards and the wings horizontal, rather than starting with the wings vertical and transitioning to have the props horizontal. Is there a dedicated tutorial for that or should I use a combination of the simple wing and rotor hover tutorials? If there isn't a dedicated tutorial, do you have any advice?

**Reply by EdoAlvarezR** [reply]
I would recommend doing all the tutorials to get familiar with how to define control inputs and maneuver kinematics. By the end of the tutorials you should have a better idea on how to simulate what you are going after, but let us know if you have more questions afterwards!
